pkg/cmd/cli/restic/server.go

```diff
@@ -68,6 +69,7 @@ func NewServerCommand(f client.Factory) *cobra.Command {
 	}
 
 	command.Flags().Var(logLevelFlag, "log-level", fmt.Sprintf("the level at which to log. Valid values are %s.", strings.Join(logLevelFlag.AllowedValues(), ", ")))
+	command.Flags().StringVar(&logFormat, "log-format", logFormat, fmt.Sprintf("the format for log output"))
```

Comment:
Let's use the `Enum` flag type here (from `pkg/cmd/util/flag`) - better UX since there are a defined set of valid values. You can find an example of using it here: https://github.com/heptio/velero/blob/master/pkg/cmd/cli/backuplocation/create.go#L63